### PR TITLE
csharp/a: add part 2

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: exercises/csharp-1-a/Csharp1ATests.cs [hunk 1]
```diff
@@ -5,13 +5,18 @@ public class SentenceTest
...
+        { "Circle Area", "Area" },
+        { "Test Your Might", "Your" },
+        { "Extremely fresh vegetables", "vegetables" }
     };
 
     [Theory]
     [MemberData(nameof(SentenceAndWordWithMostVowels))]
     public void WordWithMostVowels(string sentence, string expected) =>
-        Assert.Equal(expected, Sentence.WordWithMostVowels(sentence));
+        Assert.Equal(expected, Sentence.WordWithMostVowels(sentence), "Hallow");
```
Comment (iHiD) [inline exercises/csharp-1-a/Csharp1ATests.cs]: I don't think you want this :)

Comment (ErikSchierboom) [reply exercises/csharp-1-a/Csharp1ATests.cs]: I don't? Whoops! :D

File: exercises/csharp-2-a/README.md [hunk 2]
```diff
@@ -1 +1,29 @@
-TODO
+## Introduction
+
+This is Part 2 of our research into how people write C# differently. This exercise should take 10-15 minutes. There is no right or wrong way to approach it - just do what feels the most natural to you. Feel free to use books, Google or Stack Overflow, just like you would if you were programming normally, but please don't actively look for a solution to the problem as this will negatively affect the research.
+
+## Instructions
+
+Your task is to extend existing code with new functionality. The code you'll be working with has a method named `Sentence.WordWithMostVowels`. This method returns which word in a sentence has the most vowels (a, e, i, o, u). A sentence is defined as a [C# string][docs-string] consisting of one or more words. A word consists of one or more upper- or lowercase letters. Words are separated from each other by a single space character.
```
Comment (iHiD) [inline exercises/csharp-2-a/README.md]:
```suggestion
Your task is to extend existing code with new functionality. The code you'll be working with has a method named `Sentence.WordWithMostVowels`. This method takes a sentence and returns the word that has the most vowels (a, e, i, o, u). A sentence is defined as a [C# string][docs-string] consisting of one or more words. A word consists of one or more upper- or lowercase letters. Words are separated from each other by a single space character.
```

Just reads easier for me.

File: exercises/csharp-2-a/README.md [hunk 3]
```diff
@@ -1 +1,29 @@
...
+
+For example:
+
+```csharp
+Sentence.WordWithMostVowels("Hi there aunt jeanie")
+// Returns: "jeanie"
+```
+
+Currently, which word is returned when multiple words have the same (maximum) number of vowels is undefined. In these tie-breaker situations, the word with the most consecutive consonants (non-vowel letters) should be returned.
```
Comment (iHiD) [inline exercises/csharp-2-a/README.md]:
```suggestion
Your job is to extend this code to deal with tie-breaks - the situation where there are multiple words with the same number of vowels. This these cases the word with the most consecutive consonants (non-vowel letters) should be returned.
```

Again - just making it a little easier to parse.

Comment (ErikSchierboom) [reply exercises/csharp-2-a/README.md]: > This these cases

Should this be "In these cases"?

File: exercises/csharp-2-a/README.md [hunk 4]
```diff
@@ -1 +1,29 @@
...
+
+For example:
+
+```csharp
+Sentence.WordWithMostVowels("his own hints")
```
Comment (iHiD) [inline exercises/csharp-2-a/README.md]: Can we find a phrase with three words that have the same amount of letters, as an example?

Comment (ErikSchierboom) [reply exercises/csharp-2-a/README.md]: `his own game`?

File: exercises/csharp-2-a/README.md [hunk 5]
```diff
@@ -1 +1,29 @@
...
+Your task is to extend existing code with new functionality. The code you'll be working with has a method named `Sentence.WordWithMostVowels`. This method takes a sentence and returns the word that has the most vowels (a, e, i, o, u). A sentence is defined as a [C# string][docs-string] consisting of one or more words. A word consists of one or more upper- or lowercase letters. Words are separated from each other by a single space character.
+
+For example:
+
+```csharp
+Sentence.WordWithMostVowels("Hi there aunt jeanie")
+// Returns: "jeanie"
+```
+
+Your job is to extend this code to deal with tie-breaks - the situation where there are multiple words with the same number of vowels. In these cases the word with the most consecutive consonants (non-vowel letters) should be returned.
```
Comment (iHiD) [inline exercises/csharp-2-a/README.md]:
```suggestion
Your job is to extend this code to deal with tiebreaks - the situation where there are multiple words with the same number of vowels. In these cases the word with the most consecutive consonants (non-vowel letters) should be returned.
```

**Pull request merged**